CMakeLists.txt

```diff
@@ -24,7 +24,9 @@ set(CMAKE_CXX_STANDARD 17)
 
 add_subdirectory(third_party/LineairDB)
 set(LINEAIRDB_PLUGIN_DYNAMIC "ha_lineairdb")
-set(LINEAIRDB_SOURCES ha_lineairdb.cc ha_lineairdb_handler_miscs.cc)
+set(LINEAIRDB_SOURCES ha_lineairdb.cc ha_lineairdb.hh 
+                      lineairdb_field.cc lineairdb_field.hh 
+                      ha_lineairdb_handler_miscs.cc)
```

Comment:
do not import header files (`*.h`) as `*_SOURCES`.
use `target_include_directories` (https://cmake.org/cmake/help/latest/command/target_include_directories.html) instead.

Reply:
`target_*` functions may not work in the project of mysql plugins.
I revert this comment.